src/CSSList/CSSBlockList.php

```diff
@@ -95,7 +97,7 @@ public function getAllValues(
                     );
                 }
             }
-        } elseif ($element instanceof RuleSet) {
+        } elseif ($element instanceof RuleSet || $element instanceof DeclarationBlock) {
```

Comment:
Maybe we can have an interface that both classes implement so we can check for the interface here?

Reply:
Done.  Interestingly, this resolves the PHPStan warning (because the parameter type is now required to match the interface).